tests/testsuite.src/run_file.at

```diff
@@ -12691,7 +12691,9 @@ AT_DATA([prog1.cob], [
          ==FILE-COLSEQ== BY ==COLLATING SEQUENCE IS ASCII==
          ==KEY-COLSEQ== BY ====.
 ])
-AT_CHECK([$COMPILE -Wno-unfinished prog1.cob], [0], [], [])
+AT_CHECK([$COMPILE -Wno-unfinished prog1.cob], [0], [],
```

Comment:
please add `-Wno-others` here and below (or the appropriate new `-Wno-ignored-something`, if you for some reason want to add that), the warning is checked in syn_file.at